EIPS/eip-712.md

```diff
@@ -36,18 +34,15 @@ This EIP aims to improve the usability of off-chain message signing for use on-c
 
 <img src="https://github.com/ethereum/EIPs/blob/master/assets/eip-712/eth_sign.png?raw=true" style="padding-bottom: 20px; padding-top: 20px" width="60%" />
 
-Here we outline a scheme to encode data along with its structure which allows it to be displayed to the user for verification when signing. Below is an example of what a user could be shown when signing an EIP712 message.
+Here we outline a scheme to encode data along with its structure which allows it to be displayed to the user for verification when signing. Below is an example of what a user could be shown when signing an EIP-712 message.
```

Comment:
It's asking for a link here but it's a self-referential link which I don't think makes sense.

Reply:
Hm, I'll open an issue in the bot to decide how to handle self-references.

For now, maybe:

```suggestion
Here we outline a scheme to encode data along with its structure which allows it to be displayed to the user for verification when signing. Below is an example of what a user could be shown when signing a message according to this proposal.
```